Userland/Applications/Calculator/main.cpp

```diff
@@ -79,7 +79,9 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     constants_menu->add_action(GUI::Action::create("&Phi", TRY(Gfx::Bitmap::load_from_file("/res/icons/calculator/phi.png"sv)), [&](auto&) {
         widget->set_typed_entry(Crypto::BigFraction { Crypto::SignedBigInteger(16180339887), power });
     }));
-
+    constants_menu->add_action(GUI::Action::create("&C", TRY(Gfx::Bitmap::load_from_file("/res/icons/calculator/c.png"sv)), [&](auto&) {
+        widget->set_typed_entry(Crypto::BigFraction { Crypto::SignedBigInteger(2997924580000000000), power });
```

Comment:
You're doing extra work here, instead of manually multiplying by 10**10 and dividing by `power`, let's just not divide by anything (`BigFraction` has a constructor that default the denominator to zero):
```suggestion
        widget->set_typed_entry(Crypto::BigFraction { "299792458"_sbigint });
```